store-implementation/accumulo-store/src/test/java/uk/gov/gchq/gaffer/accumulostore/utils/LegacySupportTest.java

```diff
@@ -52,125 +54,98 @@
  */
 public class LegacySupportTest {
 
-    @TempDir
-    static File tempDir;
+    File tempDir;
+    Configuration conf;
+
+    @BeforeEach
+    public void setUp() {
+        conf = new Configuration();
+        tempDir = Files.createTempDir();
```

Comment:
This method is from Guava and is [deprecated since Java 7.](https://guava.dev/releases/snapshot-jre/api/docs/com/google/common/io/Files.html#createTempDir()) The recommendation is to use [`Files.createTempDirectory` from Java native `Files`](https://docs.oracle.com/javase/8/docs/api/java/nio/file/Files.html#createTempDirectory-java.lang.String-java.nio.file.attribute.FileAttribute...-).
```suggestion
        tempDir = Files.createTempDirectory(this.getClass().getName()).toFile();
```